packages/framework-integration-tests/src/read-models/car-model-read-model.ts

```diff
@@ -0,0 +1,15 @@
+import { Projects, ReadModel } from '@boostercloud/framework-core'
+import { ProjectionResult, UUID } from '@boostercloud/framework-types'
+import { CarModel } from '../entities/car-model'
+
+@ReadModel({
+  authorize: 'all',
+})
+export class CarModelReadModel {
+  public constructor(readonly id: UUID, readonly name: string, readonly brand: string) {}
+
+  @Projects(CarModel, 'id')
+  public static update(model: CarModel, old?: CarModelReadModel): ProjectionResult<CarModelReadModel> {
+    return new CarModelReadModel(model.id, model.name, model.brand)
+  }
```

Comment:
could we rename the function as `projectCarModel`? `update` in a read model looks to me misleading, since read models only return results without modifying the records. This function is not updating anything, it is just projecting the entities onto a GraphQL Query